models/cube.js

```diff
@@ -187,7 +187,7 @@ cubeSchema.index({
 
 const Cube = mongoose.model('Cube', cubeSchema);
 
-Cube.LAYOUT_FIELDS = '_id owner name type card_count overrideCategory categoryOverride categoryPrefixes';
+Cube.LAYOUT_FIELDS = '_id owner name type card_count overrideCategory categoryOverride categoryPrefixes image_uri';
 Cube.PREVIEW_FIELDS =
```

Comment:
`PREVIEW_FIELDS` now contains 'image_uri' twice (not sure how the database handles that)